src/quo2/components/base_tag.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.base-tag
+  (:require
+   [quo.react-native :as rn]))
+
+(defn style-container [size disabled border-color border-width background-color]
+  (merge {:height             size
+          :border-color       border-color
+          :background-color   background-color
+          :border-width       border-width
+          :border-radius      size
+          :align-items        :center
+          :justify-content    :center}
+         (when disabled
+           {:opacity 0.3})))
+
+(defn base-tag [_]
+  (fn [{:keys [id size disabled border-color border-width background-color accessibility-label
+               on-press type] :or   {size 32}} children]
+    [rn/touchable-without-feedback (merge {:disabled            disabled
+                                           :accessibility-label accessibility-label}
+                                          (when on-press
+                                            {:on-press #(on-press id)}))
+     [rn/view {:style (merge (style-container size disabled border-color border-width
+                                              background-color)
+                             (when (and (= type :icon) :emoji)
+                               {:width       size}))}
```

Comment:
`(and (= type :icon) :emoji)` ?
Did you meant to compare `:emoji` with something?

Reply:
it should be (= type :icon). I will make that change